EnRonSwanson/BangazonAPI
# product controller put/get/post/delete

Fixes #2.

Changes proposed in this pull request: 
-Product Controller PUT/POST/GET/DELETE methods 
- tested all methods
-

How to test: 
- requires 2 existing foreign keys in order to test (product type and customer)

- SQL STATEMENT FOR PRODUCT TYPE=  insert into ProductType
                                                                        values (null, "Appliance");
-make sure you have a customer in your database also

@EnRonSwanson/
